crates/router/src/core/payments/operations/payment_create.rs

```diff
@@ -514,6 +521,8 @@ impl PaymentCreate {
             statement_descriptor_name: request.statement_descriptor_name.clone(),
             statement_descriptor_suffix: request.statement_descriptor_suffix.clone(),
             metadata: metadata.map(masking::Secret::new),
+            business_country,
+            business_label,
```

Comment:
Shouldn't we also store the sub label in the payment intent?

Reply:
Sublabel can change for a payment intent for retries, if the merchant wants to retry the payment with a different Sublabel. So this field will be stored in payment attempt

Reply:
When retrieving a completed payment, which sub_label will be attached to the payment_intent? The successful payment_attempt's sub label?

Reply:
Yes, sub label will be of the payment attempt which was made recently, or in other words active payment attempt.

Reply:
Makes sense, cool.